packages/lib/server/getLuckyUser.ts

```diff
@@ -152,7 +152,7 @@ async function getUsersBasedOnWeights<
     []
   );
 
-  const bookingsOfNotAvailableUsers = await BookingRepository.getAllBookingsForRoundRobin({
+  const bookingsOfNotAvailableUsers = await BookingRepository.getBookingsForRoundRobin({
```

Comment:
this should only be the bookings of the last 28 days

Reply:
updated